src/cascadia/TerminalApp/CascadiaSettings.cpp

```diff
@@ -4,6 +4,9 @@
 #include "pch.h"
 #include <argb.h>
 #include <conattrs.hpp>
+#include <fstream>

```

Comment:
This should likely go into `LibraryIncludes.h`

Reply:
Should it only be `fstream `that goes in there, or should all 3?  Do you mind explaining what makes it/them special compared to `<argb.h>` and `<conattrs.hpp>`

Reply:
Uh, <fstream> I believe is a platform library. Like I think it comes from STL, right?

The other two are headers from our own project, so not libraries.

Reply:
Ok, I guess it's a nuance that's lost on me right now.  I see that STL, WIL, and a few other odds and ends have been included in there.  I'll just take it as it is and wait until someone who knows what they're doing tells me where to throw things :)